go-kit/metrics/provider/librato/librato.go

```diff
@@ -199,6 +203,10 @@ func New(URL *url.URL, interval time.Duration, opts ...OptionFunc) metrics.Provi
 		}
 	}
 
+	p.measurements = p.NewGauge("internal.measurements")
```

Comment:
What will the prefix and source/tags end up as for these?

Reply:
Whatever you use in the project. So if you're on probedom, `probedom.internal.measurements`. if on dyno-funnel`, `cedar.dyno-funnel.internal.measurements`. Source / etc are applied as normal.

Reply:
`internal` is a bit vague IMO, something like `go-kit` would be better.